go/fn/object.go

```diff
@@ -57,7 +57,7 @@ func (o *KubeObject) GetOrDie(ptr interface{}, fields ...string) {
 }
 
 // GetString returns the string value, if the field exist and a potential error.
-func (o *KubeObject) GetString(fields ...string) (string, bool, error) {
+func (o *SubObject) GetString(fields ...string) (string, bool, error) {
```

Comment:
Rename to `NestedString`?  But I can do that in the next PR.  I am planning to realign all the names between KubeObject and SubObject with `unstructed.Unstructured`, but haven't had a chance to re-pick up those work due to fishfooding.